docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -978,11 +978,17 @@ Many more security-relevant compiler options exist than are recommended in this
 | <span id="-Wl,-z,noexecheap">`-Wl,-z,noexecheap`</span> | Binutils 2.15 (Hardened Gentoo / PaX only ) | Hardened Gentoo / PaX specific Binutils extension[^noexecheap], not present in upstream toolchains.
 | <span id="-D_LIBCPP_ASSERT">`-D_LIBCPP_ASSERT`</span>   | libc++ 3.3.0 | Deprecated in favor of `_LIBCPP_ENABLE_HARDENED_MODE`[^libcpp_assert]
 | <span id="-D_LIBCPP_ENABLE_ASSERTIONS">`-D_LIBCPP_ENABLE_ASSERTIONS`</span> | libc++ 3.3.0 | Deprecated in favor of `_LIBCPP_ENABLE_HARDENED_MODE`[^libcpp_assert]
+| <span id="-mshstk">`-mshstk`</span>                                         | GCC 8<br/>Clang 6.0 | Enables shadow stack built-in functions[^gcc_mshstk] for x86_64 CET. CET instrumentation is controlled by [`-fcf-protection`](#-fcf-protection=full).
```

Comment:
As a non-expert and in comparison to the other entries in this table, the rationale as to why this option was not included is not clear to me. If it can be clarified concisely in the table, I'd appreciate that - if this would be too lengthy, a pointer elsewhere would be helpful.